bedrock/mozorg/redirects.py

```diff
@@ -569,4 +569,6 @@ def decider(request, **kwargs):
     redirect(r"^about/leadership/boards-of-directors/$", "/about/leadership/#boards"),
     # Issue 13672
     redirect(r"/VendorDPA/?$", "https://assets.mozilla.net/pdf/VendorDPA.pdf", re_flags="i"),
+    # Issue 13754
+    redirect(r"rise-?25/?$", "https://rise25.mozilla.org/"),
```

Comment:
this should technically start with a `^` to ensure it doesn't match any other use of `rise25` in a URL. 
```suggestion
    redirect(r"^rise-?25/?$", "https://rise25.mozilla.org/"),
```